gpt_playground/settings.py

```diff
@@ -475,8 +475,9 @@
 }
 # text-neutral text-error (pipeline run table styles here for tailwind build)
 DJANGO_TABLES2_ROW_ATTRS = {
-    "class": "border-b border-gray-200 hover:bg-gray-100",
+    "class": "border-b border-gray-200 hover:bg-gray-100 hover:cursor-pointer",
```

Comment:
Using a dynamic selector we can make this class only apply when the data attribute is present and not empty:

`data-[redirect-url]:[&:not([data-redirect-url=""])]:hover:cursor-pointer`

See:
* https://stackoverflow.com/questions/17248915/css-attribute-selector-for-existing-but-non-empty-attributes
* https://github.com/tailwindlabs/tailwindcss/discussions/11822#discussioncomment-6717861
* https://tailwindcss.com/docs/hover-focus-and-other-states#using-arbitrary-variants

Reply:
Hmm, I could not get this working unfortunately. The hover class isn't being applied. See https://github.com/dimagi/open-chat-studio/pull/342/commits/93ad98eefbdfa6c4929d48200f8bbc874a93fb17 for the change to select only elements where the redirect url is actually populated